src/containers/Rewards.jsx

```diff
@@ -61,12 +83,37 @@ const Rewards = () => {
       }
     }
 
+    const getUserPoints = async () => {
+      if (!user) return
+      try {
+        const userDoc = await applicantsRef(dbHackathonName).doc(user.uid).get()
+        if (!userDoc.exists) {
+          console.log('No user data found')
+          return
+        }
+        const userData = userDoc.data()
+        const points = userData.points || 0
```

Comment:
points are calculated from taking the events the hacker has attended from dayOf (dayOne, dayTwo and other) and then cross checking the actual event in dayOf and getting the points value -> total up all points